utils/utils.hre.ts

```diff
@@ -37,13 +37,13 @@ export async function deployNewProxy(
     instance = (await upgrades.deployImplementation(await getContractFactory(name, {}), {
       constructorArgs,
       kind: "uups",
-      unsafeAllow: ["delegatecall"],
+      unsafeAllow: ["delegatecall", "state-variable-immutable"],
     })) as string;
     console.log(`New ${name} implementation deployed @ ${instance}`);
   } else {
     const proxy = await upgrades.deployProxy(await getContractFactory(name, {}), initArgs, {
       kind: "uups",
-      unsafeAllow: ["delegatecall"], // Remove after upgrading openzeppelin-contracts-upgradeable post v4.9.3.
+      unsafeAllow: ["delegatecall", "state-variable-immutable"], // Remove after upgrading openzeppelin-contracts-upgradeable post v4.9.3.
```

Comment:
I believe this is OK since we will never actually upgrade our proxies: https://docs.openzeppelin.com/upgrades-plugins/1.x/faq#why-cant-i-use-immutable-variables

Reply:
If we keep this in the deployment utility then I think we should append it conditional on the chain being Blast?

I'm also OK with applying the exception to the Blast SpokePool contract, but would prefer to avoid unconditionally suppressing this on other deployments where we might not want this exception.

Reply:
I will follow this up with another PR.